### ISSUE TITLE
Natural Language Interface

### ISSUE THREAD

**Issue by JSeeley** [Polymarket/agents#22]
## Summary
run-autonomous-agent is a bit intimidating to me, and this might be outside the scope of the agents project, but I would love to have a natural language interface with the polymarket API where an agent carries out my request.

### Basic example
Here are a few prompt types I'd love:
1.) Cancel all orders for Steelers vs. Bills
2.) Pause orders on my Trump tweet market for 3 hours (cancel and resubmit)
3.) Show me my Open orders for all NFL markets

### Motivation
It would be great to bridge the gap between "API traders" and "button clickers". I'm an amateur programmer, but I find it hard to track down clobTokenIds to make orders or modify.

Basically a text box would be a better interface for polymarket than the clob client or web UI.
